tests/context_mt5_test.py

```diff
@@ -53,6 +53,8 @@ def test_connect_login_errors(self, mock_init, mock_login):
         with pytest.raises(PermissionError):
             self.context_with_complete_creds.connect()
 
+    # these tests no longer serve a purpose!
```

Comment:
Please remove them, if they are no longer necessary

Reply:
Actually, I disagree. This test is still relevant, it just need to be rewritten. [Here](https://github.com/BensGitHub2022/MT5-Expert-Advisor/blob/main/src/context_mt5.py#L10) in the code it is still possible to pass in a dictionary that doesn't have all of the needed values and we still need to check that this scenario is handled properly.

Reply:
I think I understand what you mean, see update